src/snaphu/_check.py

```diff
@@ -70,6 +75,74 @@ def check_dataset_shapes(
             raise ValueError(errmsg)
 
 
+def check_dataset_sizes(
+    ntiles: tuple[int, int],
+    tile_overlap: int | tuple[int, int],
+    *,
+    regrow_conncomps: bool = True,
+    single_tile_reoptimize: bool = False,
+    **datasets: InputDataset | OutputDataset,
+) -> None:
+    """
+    Ensure that one or more datasets have shape that snaphu can handle.
```

Comment:
Let's always capitalize "SNAPHU" since that's how it's stylized on the [official website](https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/).

```suggestion
    Ensure that one or more datasets have shape that SNAPHU can handle.
```